recipe/meta.yaml

```diff
@@ -51,3 +54,4 @@ extra:
   recipe-maintainers:
     - jakirkham
     - h-vetinari
+    - jjerphan
```

Comment:
Since I bring those changes in, I think it makes sense that I also help with the maintenance of the feedstock.

What do you think?